pkg/plugin/v3/scaffolds/init.go

```diff
@@ -37,7 +37,7 @@ import (
 
 const (
 	// ControllerRuntimeVersion is the kubernetes-sigs/controller-runtime version to be used in the project
-	ControllerRuntimeVersion = "v0.6.3"
+	ControllerRuntimeVersion = "v0.7.0-alpha.6"
```

Comment:
NIT: I'm not sure how do I feel with adding an alpha version here on our own library.

Reply:
Agree, this is blocked on #1746 for this but so that I could test I wanted to push my changes.

Reply:
For v3-alpha it is fine because it is an alpha stage and not the default plugin. See; https://github.com/kubernetes-sigs/kubebuilder/issues/1746